koku/masu/external/kafka_msg_handler.py

```diff
@@ -70,18 +70,21 @@ class KafkaMsgHandlerError(Exception):
     """Kafka msg handler error."""
 
 
-def divide_csv_daily(file_path: os.PathLike, manifest_id: int):
-    """
-    Split local file into daily content.
-    """
-    daily_files = []
-
+def get_data_frame(file_path: os.PathLike):
+    """read csv file into dataframe"""
     try:
-        data_frame = pd.read_csv(file_path, dtype=pd.StringDtype(storage="pyarrow"))
+        return pd.read_csv(file_path, dtype=pd.StringDtype(storage="pyarrow"), on_bad_lines="warn")
     except Exception as error:
         LOG.error(f"File {file_path} could not be parsed. Reason: {str(error)}")
         raise error
 
+
+def divide_csv_daily(file_path: os.PathLike, manifest_id: int):
+    """
+    Split local file into daily content.
+    """
+    data_frame = get_data_frame(file_path)
```

Comment:
Does this need to be a separate function? It's great to keep `divide_csv_daily` focused, but I don't know if `get_data_frame` is doing enough to justiify its existence.

Reply:
Excellent. That's a great reason.